conda/recipes/libcuml/meta.yaml

```diff
@@ -34,26 +38,40 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - cmake {{ cmake_version }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cudatoolkit ={{ cuda_version }}
-    - gmock {{ gtest_version }}
+    - cuda-version ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    - libcublas {{ cuda11_libcublas_host_version }}
+    - libcublas-dev {{ cuda11_libcublas_host_version }}
+    - libcufft {{ cuda11_libcufft_host_version }}
+    - libcufft-dev {{ cuda11_libcufft_host_version }}
+    - libcurand {{ cuda11_libcurand_host_version }}
+    - libcurand-dev {{ cuda11_libcurand_host_version }}
+    - libcusolver {{ cuda11_libcusolver_host_version }}
+    - libcusolver-dev {{ cuda11_libcusolver_host_version }}
+    - libcusparse {{ cuda11_libcusparse_host_version }}
+    - libcusparse-dev {{ cuda11_libcusparse_host_version }}
```

Comment:
CUDA 11 can we rely on the -dev packages pulling in the others? I know @bdice prefers not to make CUDA 11 changes in these PRs unless absolutely necessary, but want to check if this cleanup is possible in which case we could do it in a follow-up.

Reply:
No, CUDA 11 `-dev` packages (which are pulled from the `nvidia` channel) don't specify their dependencies on the runtime libraries properly. We must specify both `-dev` and non `-dev` libraries for CUDA 11 math libraries.

Reply:
In other words, this is correct and no action is needed.